tests/test_ab2cb.py

```diff
@@ -185,265 +200,271 @@ def abcb_from_text(text):
-                "joblo.com",
-                "kcra.com",
-                "kctv5.com",
-                "ketv.com",
-                "koat.com",
-                "koco.com",
-                "kolotv.com",
-                "kpho.com",
-                "kptv.com",
-                "ksat.com",
-                "ksbw.com",
-                "ksfy.com",
-                "ksl.com",
-                "kypost.com",
-                "kysdc.com",
-                "live5news.com",
-                "livestation.com",
-                "livestream.com",
-                "metro.us",
-                "metronews.ca",
-                "miamiherald.com",
-                "my9nj.com",
-                "myboom1029.com",
-                "mycolumbusmagic.com",
-                "mycolumbuspower.com",
-                "myfoxdetroit.com",
-                "myfoxorlando.com",
-                "myfoxphilly.com",
-                "myfoxphoenix.com",
-                "myfoxtampabay.com",
-                "nbcrightnow.com",
-                "neatorama.com",
-                "necn.com",
-                "neopets.com",
-                "news.com.au",
-                "news4jax.com",
-                "newsone.com",
-                "nintendoeverything.com",
-                "oldschoolcincy.com",
-                "own3d.tv",
-                "pagesuite-professional.co.uk",
-                "pandora.com",
-                "player.theplatform.com",
-                "ps3news.com",
-                "radio.com",
-                "radionowindy.com",
-                "rottentomatoes.com",
-                "sbsun.com",
-                "shacknews.com",
-                "sk-gaming.com",
-                "ted.com",
-                "thebeatdfw.com",
-                "theboxhouston.com",
-                "theglobeandmail.com",
-                "timesnow.tv",
-                "tv2.no",
-                "twitch.tv",
-                "universalsports.com",
-                "ustream.tv",
-                "wapt.com",
-                "washingtonpost.com",
-                "wate.com",
-                "wbaltv.com",
-                "wcvb.com",
-                "wdrb.com",
-                "wdsu.com",
-                "wflx.com",
-                "wfmz.com",
-                "wfsb.com",
-                "wgal.com",
-                "whdh.com",
-                "wired.com",
-                "wisn.com",
-                "wiznation.com",
-                "wlky.com",
-                "wlns.com",
-                "wlwt.com",
-                "wmur.com",
-                "wnem.com",
-                "wowt.com",
-                "wral.com",
-                "wsj.com",
-                "wsmv.com",
-                "wsvn.com",
-                "wtae.com",
-                "wthr.com",
-                "wxii12.com",
-                "wyff4.com",
-                "yahoo.com",
-                "youtube.com",
-                "zhiphopcleveland.com"
+                "*3news.co.nz",
```

Comment:
Just a heads up - the cosmetic rules should also get this "all subdomains" treatment (e.g. line 115)

Reply:
👍 on both.  1) Lets make sure cosmetic rules are handled better here than in ab2cb, and 2) they're not currently applied in Brave iOS (partially bc they're not currently evaluated in slim-list)

Reply:
I left those tests commented out actually; I can keep going with this and try to get `ad_urls`, `element_hiding`, and `popups` sorted out too. Even if we do switch to using adblock-rust, it would be good to leave this repo in a good state 😄 I also should probably wire this up with Jenkins

Reply:
Wired up Travis-CI 🎉